gobblin-modules/gobblin-http/src/main/java/gobblin/writer/AvroHttpWriterBuilder.java

```diff
@@ -36,4 +38,8 @@ public AvroHttpWriterBuilder fromConfig(Config config) {
     this.responseHandler = new HttpResponseHandler();
     return this;
   }
+
+  public String getLimiterKey (State state) throws NotConfiguredException {
+    return AvroHttpWriterBuilder.class.getName();
```

Comment:
I think `urlTemplate` from `State` could be the resource id, not sure if resource id has to be globally unique,